### PR TITLE
[FIX] l10n_*: Move the update of CoA in end- script

### PR DESCRIPTION
**Pull request by clbr-odoo**
We faced multiple times error because this update doing a try_loading is called in `post-` scripts (after the module is loaded) and not in `end-` scripts (when all modules are loaded).

opw-4231366

### CODE REVIEW

**Comment by aj-fuentes** [comment]
Can we merge https://github.com/odoo/odoo/pull/182706 (and related PR) first to avoid conflicts?

**Comment by william-andre** [comment]
@aj-fuentes Git is smart enough to recognize that the file has been moved if there is not too much diff

**Review by william-andre — COMMENTED** [review]
This is good, maybe we can put a warning in `try_loading` if `not self.env.registry.ready`? (to test if it's the right condition)
It would avoid issues in the future and make debugging easier

File: addons/account/models/chart_template.py [hunk 1]
```diff
@@ -140,6 +140,8 @@ def try_loading(self, template_code, company, install_demo=True):
             chart template.
         :type install_demo: bool
         """
+        if not self.env.registry.ready and not install_demo:
```
[the remark this answers is not in the document]
Comment (clbr-odoo) [reply addons/account/models/chart_template.py:143]: But it's not when loading that kind of demo: https://github.com/odoo/odoo/blob/e4a4806e5c58c599815204a73c78a228ee230613/addons/account/demo/account_demo.xml#L31
How would you like to handle that ?

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**